azure-pipelines.yml

```diff
@@ -199,6 +199,11 @@ stages:
           name: .NET Core 3 Dev
           bar: PublicDevRelease_30_Channel_Id
           storage: release/3.0
+      - dependsOn: PublishValidation
+        channel:
+          name: .NET Core 3 Release
+          bar: PublicRelease_30_Channel_Id
+          storage: release/3.0
```

Comment:
This should be unique to avoid thrashing with the dev channel, I think.

Reply:
Updated.